website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -105,3 +105,57 @@ customresourcedefinition.apiextensions.k8s.io/vaultstaticsecrets.secrets.hashico
 
 Refer to the [VSO helm chart](/vault/docs/platform/k8s/vso/helm)
  overview for a full list of supported chart values.
+
+## Installation using kustomize
+
+You can install and update your installation using `kustomize` which allows you to extend the `config/` path of the VSO repository using kustomize primitives.
+
+To install using kustomize, download and untar/unzip the latest release from the [Releases Page](https://github.com/hashicorp/vault-secrets-operator/releases).
+```shell-session
+# Example
```

Comment:
Probably don't need this comment? Since it's part of the step-by-step instructions.

```suggestion
```